tests/spdx/parser/all_formats/test_parse_from_file.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: 2022 spdx contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+import pytest
+
+from spdx.model.document import Document
+from spdx.parser.json import json_parser
+from spdx.parser.rdf import rdf_parser
+from spdx.parser.tagvalue import tagvalue_parser
+from spdx.parser.xml import xml_parser
+from spdx.parser.yaml import yaml_parser
+
+
+@pytest.mark.parametrize(
+    "parser, format_name, extension",
+    [
+        (json_parser, "JSON", ".json"),
+        (xml_parser, "XML", ".xml"),
+        (yaml_parser, "YAML", ".yaml"),
+        (rdf_parser, "Rdf", ".rdf.xml"),
+        (tagvalue_parser, "Tag", ""),
+    ],
+)
+class TestParseFromFile:
+    def test_parse_from_file_not_found(self, parser, format_name, extension):
+        with pytest.raises(FileNotFoundError) as err:
+            wrong_file_path = os.path.join(os.path.dirname(__file__), "hnjfkjsedhnflsiafg.json")
```

Comment:
You could use `extension` here. Doesn't make a difference but feels more consistent regarding the different parsers

Reply:
This feels necessary and unnecessary at the same time :D
I'll add it